spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1579,19 +1579,24 @@ def paste(self):
         text has 'CRLF' EOL chars
         """
         clipboard = QApplication.clipboard()
-        # This is here to prevent pasting mime data in the Editor.
-        # See issue 8566 for the details.
+        text = to_text_string(clipboard.text())
         if clipboard.mimeData().hasUrls():
-            return
-        else:
-            text = to_text_string(clipboard.text())
-            if len(text.splitlines()) > 1:
-                eol_chars = self.get_line_separator()
-                text = eol_chars.join((text + eol_chars).splitlines())
-                clipboard.setText(text)
-            # Standard paste
-            TextEditBaseWidget.paste(self)
-            self.document_did_change(text)
+            # Have copied file and folder urls pasted as text paths.
+            # See PR: #8644 for details.
+            urls = clipboard.mimeData().urls()
+            if all([url.isLocalFile() for url in urls]):
+                if len(urls) > 1:
+                    text = ',\n'.join('"' + url.toLocalFile().

```

Comment:
Instead of `\n` here we should use `eol_chars` to match the user preferences. You could use:

`sep_chars = ',' + self.get_line_separator()`

then `sep_chars.join(...`

Reply:
Looks like this is resolved...